packages/block-library/src/columns/editor.scss

```diff
@@ -1,13 +1,20 @@
+@mixin flex-full-height() {
+	display: flex;
+	flex-direction: column;
+	flex: 1;
+}
+
+
 // These margins make sure that nested blocks stack/overlay with the parent block chrome
 // This is sort of an experiment at making sure the editor looks as much like the end result as possible
 // Potentially the rules here can apply to all nested blocks and enable stacking, in which case it should be moved elsewhere
 // When using CSS grid, margins do not collapse on the container.
-.wp-block-columns .block-editor-block-list__layout {
```

Comment:
Why was this changed? It undoes intentional changes from #14420. 

Noting that confusion around these classnames was anticipated in https://github.com/WordPress/gutenberg/pull/14112#issuecomment-471567423 . As such, it might warrant more exploration about how we can eliminate the wrong names (`.editor-`) from code while retaining them in the built output for compatibility.